ufersa/plataforma-sabia
# About page

## Summary


Resolves #604 

## Relevant technical choices



## QA Steps


1. Open the platform home page
2. Check that the menu now has the following links: Início, Busca, Recursos (dropdown), Sobre (LP), Contato, Blog (new tab)
3. Click the "sobre" menu link
4. Check out the about page

## Checklist
